src/Grpc/HttpApi/src/Microsoft.AspNetCore.Grpc.HttpApi/Microsoft.AspNetCore.Grpc.HttpApi.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>HTTP API for gRPC ASP.NET Core</Description>
+    <PackageTags>gRPC RPC HTTP/2 REST</PackageTags>
+    <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);$(ArtifactsShippingPackagesDir)</RestoreAdditionalProjectSources>
```

Comment:
Please remove the default-matching settings here (`$(IsPackable)` and `$(IsShipping)` as well as the useless `$(RestoreAdditionalProjectSources)`. The same for many projects in this PR.